### ISSUE TITLE
How to set multiple instances

### ISSUE THREAD

**Issue by alanotmt** [facebookresearch/detectron2#1205]
I know how to create instances now, but how to add more instances together?

instances = Instances(image_size=(310, 438))
instances.set('pred_boxes',box)
instances.set('pred_classes',class)
instances.set('scores',scores)
instances.set('pred_masks', mask)

**Comment by ppwwyyxx** [comment]
what do you mean by "add more instances together"? What's the input and output you expect?

**Comment by alekseynp** [comment]
```
many_boxes = torch.tensor([[1, 2, 3, 4], [5, 6, 7, 8], [9, 0, 1, 2]])
instances.pred_boxes = many_boxes
```

**Comment by alanotmt** [comment]
> ```
> many_boxes = torch.tensor([[1, 2, 3, 4], [5, 6, 7, 8], [9, 0, 1, 2]])
> instances.pred_boxes = many_boxes
> ```

thank you,  i got the point.
